### ISSUE TITLE
Menu items deeper than first level cannot be activated with initialActiveKey

### ISSUE THREAD

**Issue by jjaakko** [iannbing/react-simple-tree-menu#106]
I am using react-simple-tree-menu version 1.1.6. I use the component as shown in the README.md of the project.

My code is as follows:
```
import React from "react";
import TreeMenu from "react-simple-tree-menu";
// import default minimal styling or your own styling
import "../node_modules/react-simple-tree-menu/dist/main.css";
// Use the default minimal UI
import "./App.css";

function App() {
  const treeData = [
    {
      key: "first-level-node-1",
      label: "Node 1 at the first level",
      nodes: [
        {
          key: "second-level-node-1",
          label: "Node 1 at the second level",
          nodes: [
            {
              key: "third-level-node-1",
              label: "Last node of the branch",
              nodes: [] // you can remove the nodes property or leave it as an empty array
            }
          ]
        }
      ]
    },
    {
      key: "first-level-node-2",
      label: "Node 2 at the first level"
    }
  ];
  return (
    <div className="App">
      <TreeMenu
        data={treeData}
        initialActiveKey="second-level-node-1"
        initialOpenNodes={["first-level-node-1"]}
      />
    </div>
  );
}

export default App;

```
In the console I am seeing the  following error:
`index.js:1375 Warning: c: key is not a prop. Trying to access it will result in undefined being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://fb.me/react-special-props)
`

Also item with key "second-level-node-1" is not activated.

I saw a similar issue here https://github.com/iannbing/react-simple-tree-menu/issues/86 but it seems that at least in the latest release there is still a problem.

Steps to reproduce:

* use create-react-app as basis
* install react-simple-tree-menu
* copy the code above to App.js

**Comment by iannbing** [comment]
Thanks for reporting! I had a PR to remove the warning and the fix will be shipped with version `1.1.7`.

Regarding `initialActiveKey`, you need to provide the complete path to the node. In your example code, you need to give `first-level-node-1/second-level-node-1`. This is to prevent duplicate keys in different levels.

**Comment by iannbing** [comment]
#108 Released version `1.1.7`

**Comment by jjaakko** [comment]
Yep, I can confirm that with the latest release the warning is gone and with your instructions the initialActiveKey works as expected. Thanks for the quick reply!
